schema/me/conversation/index.js

```diff
@@ -144,16 +158,29 @@ export const ConversationFields = {
 
   artworks: {
     type: new GraphQLList(ArtworkType),
-    resolve: conversation => {
-      const ids = []
-
-      for (const item of conversation.items) {
+    description: "Only the artworks discussed in the conversation.",
+    resolve: ({ items }) => {
+      const results = []
+      items.forEach(item => {
         if (item.item_type === "Artwork") {
-          ids.push(item.item_id)
+          results.push(item.properties)
         }
-      }
+      })
+      return results
+    },
+  },
 
-      return gravity("artworks", { ids })
+  items: {
+    type: new GraphQLList(ConversationItemType),
+    description: "The artworks and/or partner shows discussed in the conversation.",
+    resolve: ({ items }) => {
+      const results = []
+      items.forEach(item => {
+        if (item.item_type === "Artwork" || item.item_type === "PartnerShow") {
+          results.push(item.properties)
+        }
+      })
+      return results
```

Comment:
Gotcha. Maybe we should model `ConversationItem` more completely to ensure we don’t have to deprecate the `items` field in the future when we do want to use the abstracted fallbacks? E.g.

```graphql
conversation(id: 42) {
  items: {
    title
    image_urls
    item: {
      ... on Artwork {
        artists {
          name
        }
      }
      ... on Show {
        city
      }
    }
  }
}